BringgSDK.js

```diff
@@ -1252,18 +1252,20 @@ var BringgSDK = (function () {
     var onWayPointDone = function (){
       log('way point done');
 
+      if (callbacks.driverLeftCb) {
```

Comment:
maybe it will be a nicer behavior on our part to provide the waypoint id to the callback in this case? also to the log.